ql/ql/src/queries/style/docs/NonUSSpelling.ql

```diff
@@ -0,0 +1,38 @@
+/**
+ * @name Non US spelling
+ * @description QLDocs shold use US spelling.
+ * @kind problem
+ * @problem.severity warning
+ * @id ql/non-us-spelling
+ * @tags maintainability
+ * @precision very-high
+ */
+
+import ql
+
+predicate non_us_word(string wrong, string right) {
+  exists(string s |
+    wrong = s.splitAt("/", 0) and
+    right = s.splitAt("/", 1) and
+    s = ["colour/color", "authorise/authorize", "analyse/analyze"]
```

Comment:
Maybe add "modelling/modeling"

Reply:
Nice, that catches a whole lot of mistakes (a lot of which are mine :see_no_evil:).   
Could you open a PR with that addition after I've merged this one?  
We can use that new PR to fix the existing mistakes.

Reply:
Will do.